lib/crowbar/client/command/node/list.rb

```diff
@@ -73,6 +73,10 @@ def content_from(request)
                 )
               end
             end
+
+            result.sort do |x, y|
+              x.values.join <=> y.values.join
```

Comment:
Yes, please add a comment. Or you can be sure that if I come to this code and patch it, I'll have the same thought about non-deterministic order and will change the code ;-)